### PR TITLE
Release 4.4.0

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,164 @@ with advance notice in the **Deprecations** section of releases.
...
+  These hooks will be used by ``pytest-xdist``, ``pytest-subtests``, and the replacement for
+  resultlog to serialize and customize reports.
+
+  They are experimental, meaning that their details might change or even be removed
+  completely in future patch releases without warning.
+
+  Feedback is welcome from plugin authors and users alike.
+
+
+- `#4987 <https://github.com/pytest-dev/pytest/issues/4987>`_: ``Collector.repr_failure`` respects ``--tbstyle``, but only defaults to ``short`` now (with ``auto``).
```
Comment (blueyed) [inline CHANGELOG.rst]:
```suggestion
- `#4987 <https://github.com/pytest-dev/pytest/issues/4987>`_: ``Collector.repr_failure`` respects the ``--tb`` option argument, but only defaults to ``short`` now (with ``auto``).
```

Comment (nicoddemus) [reply CHANGELOG.rst]: Done

**Comment by blueyed** [comment]
Disabled Code Climate's "Pull request status updates" now.
The last time I wanted to do this it reported an actual good issue in a single PR, but this here is not very helpful.

**Comment by nicoddemus** [comment]
OK sounds good.

Other than that, if there are no other issues, please approve. 👍

**Comment by blueyed** [comment]
@nicoddemus 
Haven't looked in detail really, would appreciate if somebody else approves it.

**Comment by nicoddemus** [comment]
I have changed the target to `features` temporarily so it is easier to review. After we get approvals I will change back the target to `master` and merge. 👍

File: doc/en/example/parametrize.rst [hunk 2]
```diff
@@ -411,8 +411,6 @@ argument sets to use for each test function.  Let's run it:
         def test_equals(self, a, b):
     >       assert a == b
     E       assert 1 == 2
-    E         -1
```
Comment (nicoddemus) [inline doc/en/example/parametrize.rst]: Those changes are actually correct, they were a recent fix by how "verbose" was handled internally by some plugins (`-q` was being treated as `-v` in some cases.

**Pull request merged**